src/main/groovy/life/qbic/utils/DataParserException.groovy

```diff
@@ -0,0 +1,22 @@
+package life.qbic.utils
```

Comment:
I think we should think about the package a bit more. It might be more valuable to have a `life.qbic.datasets.parsers` package or something

Reply:
I strongly agree, I also thought about it. I think the latter domain suggestion is nailing it